include/library/spdm_lib_config.h

```diff
@@ -407,4 +407,8 @@
 #define LIBSPDM_ENABLE_MSG_LOG 1
 #endif
 
+#ifndef LIBSPDM_CHECK_MACRO
+#define LIBSPDM_CHECK_MACRO 0
```

Comment:
why disable? I think it is good to add them to find issue earlier.

Reply:
Since this check is in internal configuration header it will be `#include`d in every file and checked by the compiler. It could affect compile times.